src/components/GetStarted/Features.tsx

```diff
@@ -66,7 +66,7 @@ function Features({
           <div
             css={`
               position: relative;
-              padding-top: 106%;
+              padding-top: 136%;
```

Comment:
The previously value was to match the image minus it's bottom shadow so that alignment fits nicely. I think this is causing some layout problem on smaller screens though so I see why you updated it! but now it's making the larger screen off center.

Let's revert it to the old value for now, I think the problem isn't with the percentage, but instead with how it's width is being scaled.